contracts/hooks/examples/FullRange.sol

```diff
@@ -8,7 +8,7 @@ import {BaseHook} from "../../BaseHook.sol";
 import {SafeCast} from "@uniswap/v4-core/src/libraries/SafeCast.sol";
 import {IHooks} from "@uniswap/v4-core/src/interfaces/IHooks.sol";
 import {CurrencyLibrary, Currency} from "@uniswap/v4-core/src/types/Currency.sol";
-import {CurrencySettleTake} from "@uniswap/v4-core/src/libraries/CurrencySettleTake.sol";
+import {CurrencySettler} from "@uniswap/v4-core/test/utils/CurrencySettler.sol";
```

Comment:
Shall this be moved back to src/libraries?

Reply:
the plan is originally to redefine the library in `v4-periphery/src/libraries/CurrencySettler` but i'm OK using core's for now